database/migrations/2021_04_01_172130_drop_old_squawk_assignments_tables.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+use Carbon\Carbon;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class DropOldSquawkAssignmentsTables extends Migration
```

Comment:
Does this migration work when no assignments are present too?

Reply:
It would not, so fixed!